cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -234,14 +231,9 @@ func (sub EthereumSub) Start(completionEvent *sync.WaitGroup) {
 			// The user who starts this must provide a valid last processed block
 			if lastProcessedBlock.Cmp(big.NewInt(0)) == 0 {
 				lastProcessedBlock.Sub(endingBlock, big.NewInt(1))
-			} else if catchUpNeeded {
```

Comment:
Have you tested these changes by killing the relayer, sending a bunch of transactions, and then restarting the relayer to see if it picks up on the transactions that it misses?

Reply:
I tried the killing relayer case, and just checked at block level. Relayer can start from the lastProcessedBlock stored in level db. will try some transaction.

Reply:
So you killed it, then you advanced blocks and submitted transactions, then when you woke it back up, it processed all blocks that it missed when asleep? That needs to happen to approve this PR.

Reply:
the test case is verified. ebrelayer restart and processed transactions it missed during offline.

Reply:
That is good. We will need to funnel this through the pipeline going from swingset to sandpit to mainnet with a release.